index.html

```diff
@@ -9,20 +9,26 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <nav class="d-none mobile-nav" id="menu-nav">
+      <button id="close-btn" type="button" class="d-none menu-bars">
+        <i class="fas fa-times"></i>
+      </button>
+      <div class="menu-items">
+        <a class="menu-item" href="#projects">Portfolio</a>
+        <a class="menu-item" href="#about-me">About</a>
+        <a class="menu-item" href="#about-me">Contact</a>
+      </div>
+    </nav>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
```

Comment:
- [ ] As per the change requested by the previous reviewer, please wrap the logo in an anchor (`<a>`) tag that links to the homepage. This is in following with convention and makes for a better user experience.

Reply:
This is actually not a requirement. We only need to implement the mobile menu and not mess with any parts of the already existing page.

Reply:
Oh yeah, you are right.